src/main/kotlin/racingcarWinner/core/WinnerRacing.kt

```diff
@@ -0,0 +1,39 @@
+package racingcarWinner.core
+
+import java.lang.IllegalArgumentException
+
+class WinnerRacing {
+    var cars: List<Car> = mutableListOf()
+        private set
+
+    var maxMoveStep: Int = 0
+        private set
+
+    fun setCars(carNames: List<String>) {
+        cars = carNames.map { carName ->
+            Car(carName.trim())
+        }
+    }
+
+    fun startRacing(tryNumber: Int): Map<String, Int> {
+        if (tryNumber < INIT_TRY_NUMBER) throw IllegalArgumentException(MessageCode.TryNumberException.message)
+        val carMap = mutableMapOf<String, Int>()
+        cars.forEach { car ->
+            car.move(Util.getRandomNumber())
+            setMaxMoveStep(car.moveStep)
+            carMap[car.carName] = car.moveStep
+        }
+        return carMap
+    }
```

Comment:
```suggestion
    fun startRacing(tryNumber: Int): Map<String, Int> {
        if (tryNumber < INIT_TRY_NUMBER) throw IllegalArgumentException(MessageCode.TryNumberException.message)
        
        val carMap = mutableMapOf<String, Int>()
        
        cars.forEach { car ->
            car.move(Util.getRandomNumber())
            setMaxMoveStep(car.moveStep)
            carMap[car.carName] = car.moveStep
        }
        return carMap
    }
```

가독성을 위해 의미가 변경되는 부분에서는 한칸 띄어주는게 좋습니다 👍